taxi/tests/test_admin.py

```diff
@@ -0,0 +1,59 @@
+from django.test import TestCase
```

Comment:
There is no need to add blank line between imports that belong to the same import group